frontend/src/vendor/Segment.ts

```diff
@@ -58,8 +58,8 @@ export const initializeSegment = (): void => {
   // for methods in Analytics.js so that you never have to wait
   // for it to load to actually record data. The `method` is
   // stored as the first argument, so we can replay the data.
-  analytics.factory = function(method: any) {
-    return function(...args: any[]) {
+  analytics.factory = function (method: any) {
```

Comment:
It is expected and required by latest version of prettier:
> Always add a space after the function keyword ([#3903](https://github.com/prettier/prettier/pull/3903) by [@j-f1](https://github.com/j-f1), [@josephfrazier](https://github.com/josephfrazier), [@sosukesuzuki](https://github.com/sosukesuzuki), [@thorn0](https://github.com/thorn0); [#7516](https://github.com/prettier/prettier/pull/7516) by [@bakkot](https://github.com/bakkot))

https://prettier.io/blog/2020/03/21/2.0.0.html

Reply:
The formatting seems fine to me, but I think we want to exclude all files in this `frontend/src/vendor` directory, because it's third-party code that we don't want to modify. Maybe the `prettierrc` config "exclude" statement needs to be broader to include this vendor directory as well?